### PR TITLE
Add Faculdade Ideal Wyden

### PR DESCRIPTION
**Pull request by IlleanaPlanzo**
- Official site URL: https://www.wyden.com.br/faci
- Address: Tv. Tupinambás, 461, Belém, Brazil
- Proof of long-term (>1 year) course: On the official site (https://www.wyden.com.br/faci), go to the section "Pesquisar Cursos" (Search Courses) and select "Pará" in the first box, "Faci" in the second box, then finally the courses list will load in the third box where you can see "Ciência da Computação" (Computer Science) among the first results. Also, there's another website with the courses list and you can go directly to Computer Science with this URL: https://inscricoes.facieducacional.com.br/curso/ciencias-da-computacao
- Proof of email domain: On this URL (https://blog.wyden.com.br/noticias/central-de-contatos-faci/) you can find a list of teacher's institucional emails with the domain. On the official site (https://www.wyden.com.br/faci), go to the top bar menu and select "Blog Universitário" to access this second blog/website.

### CODE REVIEW

**Comment by philipto** [comment]
@IlleanaPlanzo Could you please tell me where I can read more about Wyden? It seems to be a large network of schools. JetBrains doesn't generally add umbrella institutions to this repository. I need to understand exactly what the affiliation with Wyden means, and then add this particular email domain to the repository. Thank you.

**Comment by IlleanaPlanzo** [comment]
Hi @philipto, 
Wyden Educacional is a educational group here in Brazil. You can see the Wyden schools across the country in this page of the official site: https://www.wyden.com.br/instituicoes/instituicoes-wyden
The Wyden Educacional group was owned by the Adtalem Brasil Holding, witch was bought recently by YDUQS corporation, as you can read here: https://www.businesswire.com/news/home/20191021005193/en/Adtalem-Global-Education-Reaches-Agreement-to-Sell-Adtalem-Educacional-do-Brasil-to-YDUQS-in-a-Deal-Valued-At-465M
In summary, my school (Faculdade Ideal Wyden) is one of the Wyden Educational schools.
Thanks for getting back so quickly, have a good evening!

**Comment by philipto** [comment]
@IlleanaPlanzo Thank you very much for the clarification. Pull request merged.
